src/styles/blocks/header.scss

```diff
@@ -0,0 +1,59 @@
+$main-color: #060b35;
+$hover--color: #00acdc;
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 2px 4px #0000000d;
+
+  &__logo {
+    display: flex;
+  }
+}
+
+a {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
Don't use tag names for styling (except ``html`` and ``body``)

![image](https://github.com/mate-academy/layout_catalog/assets/54860641/a64f4cb8-19b4-4d81-84b1-d888797ef6ee)